shlr/sdb/include/sdb/r_vec.h

```diff
@@ -0,0 +1,344 @@
+#ifndef R2_VEC_H
+#define R2_VEC_H
+
+#include <r_util/r_assert.h>
+#include <r_types.h>
+#include <stdlib.h>
```

Comment:
i think stdlib and stdbool are already included in r_types.

Reply:
Ah nice.